deploy/static-sites.txt

```diff
@@ -1,2 +1,3 @@
 documentation/**
-textbook/**
\ No newline at end of file
+textbook/**
+ecosystem/**
```

Comment:
So the one thing to check here is if this will conflict with updating https://qiskit.org/ecosystem/ from this repo (AFAIK that page is still managed here). This file gets used with `--exclude-from` in rclone which means this will not sync any files which match any expression here. This is the correct syntax to expand external (relative to this repo) of the qiskit.org/ecosystem path, because we use `rclone sync` to ensure deleted paths from the build get removed on the host, this is necessary to enable usage of the subdir outside of this repo as otherwise rclone would delete externally managed files as their not in the source side of the sync. But we should check if this doesn't block updates to the existing page, and if it does if we can adjust this filter to enable updates from here but also not blocking external usage of the subdir.

The docs on filtering in rclone are here:  https://rclone.org/filtering/